core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -28,7 +28,7 @@
       </multiple-choice-editor>
 
       <!-- TODO(sll): Update this condition when control of individual field visibility is added. -->
-      <div ng-if="!$last">
+      <div ng-if="!$last" id="getSubfieldId(question, 'Bridge text')">
```

Comment:
Examine it in the Elements tab in Chrome. Does it actually get interpolated, or does it stay as the literal text `getSubfieldId...`?